src/current/cockroachcloud/authorization.md

```diff
@@ -146,100 +117,56 @@ Users with this role can view cluster details, allowing them to [export a connec
+- [Create or delete a cluster](create-your-cluster.html)
+- [Invite team members to the organization](managing-access.html#invite-team-members-to-an-organization)
+- [Manage an organization's users and their roles](managing-access.html#manage-an-organizations-users)
+- [Create and manage SQL users](managing-access.html#create-a-sql-user)
+- [Manage billing for the organization](billing-management.html)
+- [Restore databases and tables from a {{ site.data.products.db }} backup](use-managed-service-backups.html#ways-to-restore-data)
+- [Delete an organization](managing-access.html#delete-an-organization)
 
 {{site.data.alerts.callout_info}}
-Org Administrator (legacy) role will be deprecated in favor of more fine-grained roles for separately administering organization-level user-management functions, cluster management functions, and billing management functions, once those fine-grained roles are [generally available (GA)](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html).
-{{site.data.alerts.end}}
-
-### Is it possible to assign more than one role to a user in a {{ site.data.products.db }} organization?
-
-Yes, it is possible, and often necessary, to assign more than one role to a user. The default minimum access role Org Member is always assigned to every user as long as they’re a part of the {{ site.data.products.db }} organization. Beyond that, every other assigned role is additive to the overall entitlements of a user. Best example of this is the initial user who is by default assigned the Org Member, Org Administrator (legacy), and Cluster Admin roles when they create the {{ site.data.products.db }} organization. 
-
-### Can we follow the least privilege principle by using the roles available in the {{ site.data.products.db }} authorization model?
+This role will is deprecated in favor of the following more fine-grained roles, which, in combination, cover the same permissions:
 
-Yes, the roles available in the {{ site.data.products.db }} authorization model allow admins to grant only those entitlements to users that are supposed to map to their intended workflows. 
+- [Org Administrator](#org-administrator)
+- [Cluster Administrator](#cluster-administrator)
+- [Billing Coordinator](#billing-coordinator)
+{{site.data.alerts.end}}
+  
+### Org Developer (legacy)
 
-Cluster level roles like Cluster Admin or Cluster Developer allow to perform pertinent actions for one or more clusters, while providing differentiation between admin and non-admin entitlements. Whereas, the Organization level roles like Org Administrator (legacy), Org Developer (legacy) allow admin and non-admin access respectively for the entire organization.
+Org Developer (legacy) can read high-level information for all clusters, and monitor all clusters using DB Console.
```

Comment:
Not necessarily something for this PR, what does "high-level information" include?

Reply:
absolutely nothing.